pallets/free-calls/src/lib.rs

```diff
@@ -28,6 +28,12 @@ pub use pallet::*;
 #[cfg(test)]
 mod mock;
 
+#[cfg(test)]
+mod test_pallet;
+
+#[cfg(test)]
+mod tests;
```

Comment:
Looks like config validation function is not covered with tests:
https://github.com/dappforce/subsocial-node/blob/430fe055892bfdda1b328af81b5f9792a2ee559d/pallets/free-calls/src/config.rs#L72